### PR TITLE
feat: document how to add swag opportunities

### LINKED ISSUE #512: Extend Contributing.MD
- [x] I've checked that this isn't a duplicate issue.

What do you think of adding a simple section on how to add a swag opportunity to the Contributing file? I could do it and it would be pretty small but helpful for people not experienced with the technologies used as well as the project.

### PR DESCRIPTION
**Pull request by sampoder**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

This will make it easier for people to understand how to add swag. Resolves #512

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -16,3 +16,10 @@ You might be wondering what a contribution is. Here are some things that you can
 - Review [open Pull Requests](https://github.com/swapagarwal/swag-for-dev/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc).
 
 Thanks again, we hope to see you soon! ❤
+
+## How do I add a Swag Opportunity?
+
+1. Open an issue [here](https://github.com/swapagarwal/swag-for-dev/issues/new?template=----new-swag-opportunity.md)
```
Comment (aslafy-z) [inline CONTRIBUTING.md]: Or implement one that already exists [insert link to new-swag issues list]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,3 +16,10 @@ You might be wondering what a contribution is. Here are some things that you can
...
+2. Fork the Repo
```
Comment (aslafy-z) [inline CONTRIBUTING.md]:
```suggestion
2. Fork the repository
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,3 +16,10 @@ You might be wondering what a contribution is. Here are some things that you can
...
+3. Edit data.json to add your swag opportunity. Please make it in alphabetical order as well as using the data you first posted the issue about the swag opportunity
```
Comment (aslafy-z) [inline CONTRIBUTING.md]: Adapt to match the first step comment I made

Comment (aslafy-z) [reply CONTRIBUTING.md]: And magic quote the filename

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,3 +16,10 @@ You might be wondering what a contribution is. Here are some things that you can
...
+4. Open a pull request!
```
Comment (aslafy-z) [inline CONTRIBUTING.md]: Mention that all the template has to be filled out (including original issue number)

**Comment by aslafy-z** [comment]
Thank you @sampoder, we were missing this part! I did few comments, please review them.

**Comment by sampoder** [comment]
Fixed!

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -16,3 +16,11 @@ You might be wondering what a contribution is. Here are some things that you can
 - Review [open Pull Requests](https://github.com/swapagarwal/swag-for-dev/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc).
 
 Thanks again, we hope to see you soon! ❤
+
+## How do I add a Swag Opportunity?
+
+1. Open an issue [here](https://github.com/swapagarwal/swag-for-dev/issues/new?template=----new-swag-opportunity.md) or  implement one [that already exists](https://github.com/swapagarwal/swag-for-dev/labels/%3Atada%3A%20new%20swag)
```
Comment (vikaspotluri123) [inline CONTRIBUTING.md]:
```suggestion
1. Open an issue [here](https://github.com/swapagarwal/swag-for-dev/issues/new?template=----new-swag-opportunity.md) or implement one [that already exists](https://github.com/swapagarwal/swag-for-dev/labels/%3Atada%3A%20new%20swag) 
```

Double space 😄

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -16,3 +16,11 @@ You might be wondering what a contribution is. Here are some things that you can
 - Review [open Pull Requests](https://github.com/swapagarwal/swag-for-dev/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc).
 
 Thanks again, we hope to see you soon! ❤
+
+## How do I add a Swag Opportunity?
+
+1. Open an issue [here](https://github.com/swapagarwal/swag-for-dev/issues/new?template=----new-swag-opportunity.md) or  implement one [that already exists](https://github.com/swapagarwal/swag-for-dev/labels/%3Atada%3A%20new%20swag) 
+2. Fork the repository
+3. Edit [`data.json`](data.json) to add your swag opportunity. Please make it in alphabetical order as well as using the data the issue was first posted.
```
Comment (vikaspotluri123) [inline CONTRIBUTING.md]:
```suggestion
3. Edit [`data.json`](data.json) to add your swag opportunity. Please keep it in alphabetical order, and use the date the issue was first posted.
```

**Comment by sampoder** [comment]
@vikaspotluri123 fixed!

**Comment by vikaspotluri123** [comment]
@aslafy-z we need to push a fix for the missing Umbraco swag image before merging 🤔

**Comment by sampoder** [comment]
@vikaspotluri123 Does that fix this?

**Comment by vikaspotluri123** [comment]
Yep!

**Pull request merged**